client/app/shared/forms/form.js

```diff
@@ -0,0 +1,37 @@
+import BackboneForm from 'backbone-forms/distribution/backbone-forms.js';
+import templates from './templates';
+import _ from 'underscore';
+
+BackboneForm.validators.phone = function(options) {
+  options = _.extend({
+    type: 'phone',
+    message: 'Invalid phone number.',
+    // Taken from http://stackoverflow.com/questions/123559/a-comprehensive-regex-for-phone-number-validation
+    regexp: /^(?:(?:\+?1\s*(?:[.-]\s*)?)?(?:\(\s*([2-9]1[02-9]|[2-9][02-8]1|[2-9][02-8][02-9])\s*\)|([2-9]1[02-9]|[2-9][02-8]1|[2-9][02-8][02-9]))\s*(?:[.-]\s*)?)?([2-9]1[02-9]|[2-9][02-9]1|[2-9][02-9]{2})\s*(?:[.-]\s*)?([0-9]{4})(?:\s*(?:#|x\.?|ext\.?|extension)\s*(\d+))?$/i
+  }, options);
```

Comment:
This ( apparently ) is (xxx) xxx-xxxx only. We should change that.  We'll let it go for now, and I'll open a ticket once this is accepted.